strings-csharp/ReplaceSpecialCharactersInString/ReplaceSpecialCharactersInStringTests/ReplaceSpecialCharactersInString.cs

```diff
@@ -0,0 +1,89 @@
+using ReplaceSpecialCharactersInString;
+
+namespace ReplaceSpecialCharactersInStringTests
+{
+    [TestClass]
+    public class ReplaceSpecialCharactersInStringTests
+    {
+        private const string OriginalString = "a*b*c*d*e*f*g#h#i#j*k*l*m*n*o*p*q*r*s*t*u*v*w#x#y#z";
+
+        private const string ExpectedOutput = "a*b*c*d*e*f*g h i j*k*l*m*n*o*p*q*r*s*t*u*v*w x y z";
+
+        private char OldChar { get; set; } = '#';
```

Comment:
Why the other two are const but OldChar and NewChar are not?
Why the declaration is different, {get;set;} in OldChar and NewChar?